website/docs/intro-weave-gitops.mdx

```diff
@@ -11,27 +11,26 @@ hide_title: true
 
 Today Weave GitOps defaults are Flux, Kustomize, Helm, SOPS, and Kubernetes Cluster API. If you use Flux already, then you can easily add Weave GitOps to create a platform management overlay.
 
-:::tip 
+:::tip
 Adopting GitOps can bring a number of key benefits—including faster and more frequent deployments, easy recovery from failures, and improved security and auditabiity. Check out our [GitOps for Absolute Beginners](https://go.weave.works/WebContent-EB-GitOps-for-Beginners.html) eBook and [Guide to GitOps](https://www.weave.works/technologies/gitops/) for more information.
 :::
 
-## Overview
+## Getting Started
 
-This user guide provides content that will help you to [install](./installation/index.mdx) and [get started](./getting-started/intro.mdx) with our free and paid offerings:
-- **Weave GitOps Open Source**: a simple, open source developer platform for people who don't have Kubernetes expertise but who want cloud native applications. It includes the UI and many other features that take your team beyond a simple CI/CD system. Experience how easy it is to enable GitOps and run your apps in a cluster. 
-- **Weave GitOps Enterprise**: an [enterprise version](./intro-ee.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team.
+This user guide provides content that will help you to install and get started with our free and paid offerings:
+- **Weave GitOps Open Source**: a simple, open source developer platform for people who don't have Kubernetes expertise but who want cloud native applications. It includes the UI and many other features that take your team beyond a simple CI/CD system. Experience how easy it is to enable GitOps and run your apps in a cluster. [Go here to install](./open-source/getting-started/install-OSS.mdx).
+- **Weave GitOps Enterprise**: an [enterprise version](./enterprise/getting-started/intro-enterprise.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team. Note that Enterprise offers a more robust UI than what you'll find in our open source version. [Go here to install](./enterprise/getting-started/install-enterprise.mdx).
 
-:::tip Want to learn more about how <Link to="../intro-ee">Weave GitOps Enterprise</Link> can help your team?
+:::tip
+Want to learn more about how <Link to="./enterprise/getting-started/intro-enterprise.mdx">Weave GitOps Enterprise</Link> can help your team?
```

Comment:
This tip looks odd https://staging.docs.gitops.weave.works/upstream-doc-changes-june2/docs/next/intro-weave-gitops/#getting-started
Perhaps we need to remove the <Link> from there?

Reply:
I tried fixing that but yeah, the problem persists. @luizbafilho, wdyt?

Reply:
Thank you @luizbafilho  -- I applied that suggestion, hopefully it works now.

Reply:
Looks much better now 👍

Reply:
yayyyyyy